### PR TITLE
test coverage for public views

### PR DESCRIPTION
**Pull request by thraxil**

### CODE REVIEW

File: ccdb/law/tests/test_views.py [hunk 1]
```diff
@@ -17,6 +18,10 @@ def test_index(self):
         response = self.c.get("/")
         self.assertEquals(response.status_code, 200)
 
+    def test_feedback(self):
+        response = self.c.get("/")
```
Comment (sdreher) [inline ccdb/law/tests/test_views.py]: Should the url differ from test_index here?

File: ccdb/law/tests/test_views.py [hunk 2]
```diff
@@ -31,9 +36,71 @@ def test_empty(self):
...
-            status="vetted")
+        self.s = SnapshotFactory()
         r = self.c.get("/autocomplete/?term=nothing")
         self.assertEquals(r.status_code, 200)
         self.assertEquals(r.content, '[]')
+
+
+class TestViewCharge(TestCase):
+    def setUp(self):
+        self.c = Client()
```
Comment (sdreher) [inline ccdb/law/tests/test_views.py]: Just a note: by default, TestCase has a self.client attribute. Is there any need to create another?

File: ccdb/law/tests/test_views.py (hunk 1 above)
Comment (thraxil) [inline ccdb/law/tests/test_views.py]: Yeah, that might help :)

File: ccdb/law/tests/test_views.py (hunk 2 above)
Comment (thraxil) [inline ccdb/law/tests/test_views.py]: Cool. I had no idea.

**Pull request merged**